source/gmx/2020.2/CMakeLists.txt.patch

```diff
@@ -0,0 +1,29 @@
+--- /data1/anguse/yingze/gromacs-2020.2/CMakeLists.txt	2021-09-16 00:25:17.125620740 +0800
++++ CMakeLists.txt	2021-09-16 14:54:13.434223510 +0800
+@@ -134,6 +134,26 @@
+ #    (i.e., something that is exposed in installed headers).
+ set(GMX_PUBLIC_LIBRARIES "")
+ 
++# DeepMD
++message(STATUS "Compling with DeepMD...")
++add_definitions(-w) # close warning
++include_directories(${GMX_DEEPMD_ROOT}/include)
++include_directories(${GMX_TENSORFLOW_ROOT}/include)
++link_directories(${GMX_DEEPMD_ROOT}/lib)
++link_directories(${GMX_TENSORFLOW_ROOT}/lib)
++# define high precision
++add_definitions("-D HIGH_PREC")
```

Comment:
question: this means it only supports models with double precision?

Reply:
Yes. That's because we have not tested on whether single precision is enough for a gromacs simulation...And @denghuilu told me that the single-precision version is not quite stable yet due to lack of some unittests

Reply:
If so, I think you should mention this thing.

Reply:
OK.